install.sh

```diff
@@ -0,0 +1,212 @@
+#!/bin/bash
+
+# Made by Innovative Inventor at https://github.com/innovativeinventor.
+# If you like this code, star it on GitHub!
+# Contributions are always welcome.
+
+# MIT License
+# Copyright (c) 2017 InnovativeInventor
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+while [[ $# -gt 0 ]]
+do
+key="$1"
+
+case $key in
+    -p|--package)
+    package="$2"
+    specifypackage=YES
+    shift # past argument
+    shift # past value
+    ;;
+    -o|--override)
+    OSTYPE="$2"
+    shift # past argument
+    shift # past value
+    ;;
+    -c|--command)
+    command="$2"
+    shift # past argument
+    shift # past value
+    ;;
+    -b|--build)
+    build=YES
+    shift # past argument
+    ;;
+    -h|--help)
+    help=YES
+    shift # past argument
+    ;;
+esac
+done
+
+display_help() {
+    echo
+    echo 'An tool for installing or building for LanguageTool.'
+    echo 'Usage: install.sh <option> <package>'
+    echo 'Options:'
+    echo '   -h --help                   Show help'
+    echo '   -o --override <OS>          Override automatic OS detection with <OS>'
+    echo '   -b --build                  Builds packages from the bleeding edge development copy of LanguageTool'
+    echo '   -p --package <package>      Specifies package to install when building (default all)'
+    echo '   -c --command <command>      Specifies post-installation command to run (default gui)'
+    echo
+    echo 'Commands:'
```

Comment:
It would be easier to follow if "Commands" instructions are shown after "Package" instructions, like in the former list.